modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -102,6 +102,11 @@ class SeqTranslate(site: Site) {
 
   }
 
+  def instrumentResources(inst: Instrument): Set[Resource] = inst match {
+    case Flamingos2(_) => Set(Resource.Mount, Resource.ScienceFold, Resource.F2)
```

Comment:
Does this mean that to use flamingos2 you need the `Mount`, `ScienceFold` and `F2` available?
Is the check done when it is constructed on when it runs?